examples/Bunch/include/simulation_defines/param/memory.param

```diff
@@ -53,4 +53,8 @@ constexpr uint32_t BYTES_EXCHANGE_Y = 6 * 512 * 1024; //6 MiB
 constexpr uint32_t BYTES_EXCHANGE_Z = 4 * 256 * 1024; //4 MiB
```

Comment:
I can do that if you want, after yours is through - if you don't wanna rebase. But since this is such a minor change which only affects the comments I'd also be fine if it happened in the same PR.

There were some more typos in parts of the code that you didn't specifically touch in the files. [Here](https://github.com/ComputationalRadiationPhysics/picongpu/pull/1703/files#diff-685d4f0ffd9e2a0bfe7283c093e47544R156) to be precise, and in the comment below where it says *negativ* instead of *negative*. But a separate commit would be enough for this.